docs/DeveloperGuide.adoc

```diff
@@ -279,87 +279,362 @@ Refer to the guide <<DevOps#, here>>.
+*Priorities*
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+* High (must have) - `* * *`
+* Medium (nice to have) - `* *`
+* Low (not useful) - `*`
+
+[width="80%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a ... |I want to ... |So that I ca ...
+|`* * *` | frequent traveler | add a travel entry with weather, location and time metadata | document my past travels
+
+|`* * *` | traveler always on the go | record short text entries recording my travels, similar to that of a Tweet on Twitter | can keep track of my activities without excessive typing
+
+|`* * *` |user |add an activity into an itinerary list |see my planned activities at a glance and better plan my trip
+
+|`* * *` |user |tag a location to each activity in the itinerary |view where each activity is taking place
+
+|`* * *` |user |edit the entries in my itinerary list |can amend any mistakes when entering data, or change my travel plans
+
+|`* * *` |user | see a chronological timeline of the activities in the itinerary | can have a clear idea of my travel plans and schedule dur
+
+|`* * *` |frequent traveler |see a record of all my past itineraries |can review my past travels
+
+|`* * *` |traveler always meeting new people overseas |add a contact that I met during my trip with metadata including their name, age, phone number, location where I met them, and their country of origin |can keep in contact with them
+
+|`* * *` |user |delete contacts |can get rid of contacts that I have not been in touch with for some time
 
-|`* * *` |user |add a new person |
+|`* * *` |user |view my itineraries, contact lists and journals specific to the trip they are relevant to |can effectively plan for multiple trips simultaneously
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |customize trip names |can distinguish between different trips that happen in the same location within similar date ranges
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* *` |frequent traveler |to be able to see a timeline view of all my short journal entries and photos, including the location and time of the photos and entries, per trip| ??
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |food lover |add a meal with description, photo, price and location |keep track of what I ate
+
+|`* *` |user |sort my travel entries by category depending on the ‘feeling’ tag of each travel entry |see my best moments
+
+|`* *` |busy user |list and record items in a packing list |easily reference what I need for a trip at a glance
```

Comment:
It might be better to split this into two different use cases (follow Single Responsibility Principle of User Stories)

So for example

```
| busy user |list items ... |
|  busy user | record items ... |
```